doc/source/contributing_docstring.rst

```diff
@@ -189,31 +189,34 @@ infinitive verb.
 
 .. code-block:: python
 
-    def astype(dtype):
+    def astype(dtype):    # noqa: F811
```

Comment:
At that position no, I went for symmetry. Indeed using `noqa` is quite confusing and therefore I try to avoid it. 
@WillAyd At this position I thought putting every method in an extra .. code-block:: would be the best option, however it changes the documentation quite signifikant?!